### PR TITLE
csi: Fix socket extension comment

### PR DESCRIPTION
**Pull request by endocrimes**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

The CSI socket has used a sock extension since 179d8e108e8. This removes
the stale TODO for updating it.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/volume/csi/csi_plugin.go [hunk 1]
```diff
@@ -51,8 +51,6 @@ const (
 
 	// TODO (vladimirvivien) implement a more dynamic way to discover
 	// the unix domain socket path for each installed csi driver.
-	// TODO (vladimirvivien) would be nice to name socket with a .sock extension
-	// for consistency.
 	csiAddrTemplate = "/var/lib/kubelet/plugins/%v/csi.sock"
```
Comment (vladimirvivien) [inline pkg/volume/csi/csi_plugin.go]: I believe this is no longer used as well, that line can also be removed.

Comment (vladimirvivien) [reply pkg/volume/csi/csi_plugin.go]: Referring to the `csiAddrTemplate` variable here.

Comment (endocrimes) [reply pkg/volume/csi/csi_plugin.go]: hmm i'll give it a git-grep later :+1:

**Comment by endocrimes** [comment]
Just removed the unused var too :+1:

Sorry for taking a while - was mostly heads down on unrelated things.

**Comment by vladimirvivien** [comment]
/lgtm

Thanks

**Pull request merged**